earthpy/tests/test_io.py

```diff
@@ -1,34 +1,40 @@
-""" Tests for example data. """
+""" Tests for io module. """
 
 import os
+import requests
 import pytest
 import numpy as np
 import rasterio as rio
 import geopandas as gpd
-from earthpy.io import path_to_example
+import earthpy.io as eio
+
+
+@pytest.fixture
+def eld(tmpdir):
```

Comment:
I thought it was more explicit to call the class being tested by name directly in these tests, so that readers don't have to retrace where `earthpy.data` comes from (by noticing that there is not an `earthpy/data.py` module, and then discovering `earthpy.data` comes from `earthpy/__init__.py`). 

We could add `import earthpy as et`, and then reset the `earthpy.data.path` attribute, but that seemed a less direct than creating the fixture directly from the class.

Reply:
ok sure! that makes perfect sense for the tests :) thanks!